fragments/csi/ceph/bundle.yaml

```diff
@@ -0,0 +1,15 @@
+# This is an incomplete bundle fragment. Do not attempt to deploy.
+# the Ceph-csi charm is not completed, it is a WIP.
+services:
+  "ceph-csi":
+    charm: "cs:~containers/ceph-csi"
+    annotations:
+      "gui-x": "450"
+      "gui-y": "750"
+relations:
+  - - "ceph-csi:etcd"
+    - "etcd:db"
+  - - "ceph-csi:csi"
+    - "kubernetes-master:csi"
```

Comment:
This looks strange. Do kubernetes master and worker have this relation? How will this work? I might be losing the big picture.

Reply:
Any update on this?

Reply:
I am not aware of any plans to add the csi interface so it is a -1 from me on this PR. We should not leave the bundle builder broken in the prospect that at some point in the future we might have a csi interface. Never seen a design document for this interface, I do not know what this interface is supposed to do.

My suggestion is to put the CSI parts in a separate PR and pair it with a top level design document and trello card with what needs to be done across the entire set of charms and bundles.

Reply:
Thank you @CalvinHartwell can you also please update the PR's title so we remember we did not merge any CSI related things with this PR?